relay-metrics/src/aggregation.rs

```diff
@@ -2128,68 +2174,68 @@ mod tests {
     #[test]
     fn test_parse_buckets() {
         let json = r#"[
-          {
-            "name": "endpoint.response_time",
-            "unit": "millisecond",
-            "value": [36, 49, 57, 68],
-            "type": "d",
-            "timestamp": 1615889440,
-            "width": 10,
-            "tags": {
-                "route": "user_index"
-            }
-          }
-        ]"#;
+           {
```

Comment:
Please revert indentation changes. These indents are now 3 + 2 spaces, which is inconsistent.